packages/picasso.js/src/core/chart-components/legend-seq/index.js

```diff
@@ -1,5 +1,11 @@
 import legendSeq from './legend-seq';
 
+/**
+ * @typedef {object} ComponentLegendSeq
+ * @extends ComponentSettings
+ * @property {string} scale Reference to sequential color scale
+ */
```

Comment:
missing?
@property {ComponentLegendSeq.setting} settings Legend settings
+
example?

Reply:
`ComponentLegendSeq.settings` is definied in `legend-seq.js`?

Reply:
the type `ComponentLegendSeq.settings` is definied in `legend-seq.js`
but I can not find that it defines that an object of type ComponentLegendSeq has a `settings` property (of type `ComponentLegendSeq.settings`). but maybe it is there and I could just not find it.

Reply:
Yes so I believe you don't have to define the property as well